README.md

```diff
@@ -1,58 +1,78 @@
-# Sonarr
+<p align="center">
+<img src="./Logo/256.png" alt="Sonarr">
+</p>
 
 Sonarr is a PVR for Usenet and BitTorrent users. It can monitor multiple RSS feeds for new episodes of your favorite shows and will grab, sort and rename them. It can also be configured to automatically upgrade the quality of files already downloaded when a better quality format becomes available.
 
-## Major Features Include:
+## Getting Started
 
-* Support for major platforms: Windows, Linux, macOS, Raspberry Pi, etc.
-* Automatically detects new episodes
-* Can scan your existing library and download any missing episodes
-* Can watch for better quality of the episodes you already have and do an automatic upgrade. *eg. from DVD to Blu-Ray*
-* Automatic failed download handling will try another release if one fails
-* Manual search so you can pick any release or to see why a release was not downloaded automatically
-* Fully configurable episode renaming
-* Full integration with SABnzbd and NZBGet
-* Full integration with Kodi, Plex (notification, library update, metadata)
-* Full support for specials and multi-episode releases
-* And a beautiful UI
+- [Installation](https://github.com/Sonarr/Sonarr/wiki/Installation)
+- [Docker](https://github.com/Sonarr/Sonarr/wiki/Installation#Docker)
+- [FAQ](https://github.com/Sonarr/Sonarr/wiki/FAQ)
+- [Wiki](https://github.com/Sonarr/Sonarr/wiki)
+
+## Downloads
+
+Type | Releases |
+|---|---|
+Binary Releases | [GitHub Releases](https://github.com/Sonarr/Sonarr/releases)
+Docker (linuxserver.io, x86_64, arm64, armhf) | [Docker Releases](https://store.docker.com/community/images/linuxserver/sonarr)
+
+
+## Support
+
+- [Donate](https://sonarr.tv/donate)
+- [Discord](https://discord.gg/M6BvZn5)
+- [Reddit](https://www.reddit.com/r/sonarr)
+- [API v2](https://github.com/Sonarr/Sonarr/wiki/API)
```

Comment:
I can't help but feel this doesn't fit, but not sure where it should go. Perhaps under getting started as `API Documentation`?